pgqd.yaml

```diff
@@ -0,0 +1,163 @@
+package:
+  name: pgqd
+  version: "3.5"
+  epoch: 0
+  description: PgQ maintenance daemon
+  copyright:
+    - license: ISC
+  dependencies:
+    runtime:
+      - libevent
+      - py3-docutils
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - libevent-dev
+      - libtool
+      - pkgconf-dev
+      - postgresql-${{vars.pg-version}}-dev
+      - py3-docutils
+
+vars:
+  pg-version: "17"
```

Comment:
I might be lacking context here, is this version streamed? or maybe we can have a comment on why pg17?